README.md

```diff
@@ -9,16 +9,26 @@ When the library server is down, the following needs to happen:
 
 If the library server is down:
 Steps to take if the underlying database is down also (can you get to the PostgreSQL database)?
+
 1.	Move themes/unl/templates/down_no_db.html to replace header.phtml
+
 2.	Disable the UNLSierra driver and enable the NoILS driver in local/config/vufind/config.ini
+
 3.	Check the local/languages/en.ini file for the ils_offline_status message, as well as ils_offline_holdings_message and ils_offline_home_message . Make sure they say what you want.
+
 4.	Make sure in the file module/VufindUNL/src/VufindUNL/View/Helper/Root/ProxyUrl.php  __invoke function has the proxy rewrite removal in it.
+
 5.	 In themes/unl/templates/RecordDriver/SolrDefault/result-list.phtml  around line 118, uncomment the div with the class callnumAndLocation_noILS. (or cp result-list_nodb.phtml to result-list.phtml )
+
 6.	Disable any cron jobs that harvest the catalog
 
 If the database is available:
+
 1.	Move themes/unl/templates/down_db_avail.html to replace header.phtml
+
 2.	Make sure in the file module/VufindUNL/src/VufindUNL/View/Helper/Root/ProxyUrl.php  __invoke function has the proxy rewrite removal in it.
```

Comment:
May not be necessary anymore with the use of EZproxy.  Maybe a different note would be better here to check the proxy is correct?